app/channels/sketch_channel.rb

```diff
@@ -20,7 +20,7 @@ def unsubscribed
   def blink data
     if board = Board.find_by(mac: params[:mac])
       Log.received "Received input from #{board.name}<#{board.mac}>"
-      InputBroadcastJob.perform_now data, board
+      InputBroadcastJob.new.perform board
```

Comment:
asta cred ca e una din schimbarile alea care a ramas la mine pe local de cand am facut noi atunci. Ca nu imi amintesc sa fi schimbat eu asta cu .new.perform. plus ca ar fi .perform_later ca perform_async da eroare de syntaxa din cate stiu pt ca e functia care se apeleaza under the hood in sidekiq.